### ISSUE TITLE
Use `latest` for Ubuntu 20.04 1ES pools

### ISSUE THREAD

**Issue by danieljurek** [Azure/azure-sdk-tools#4127]
Pool version was pinned to mitigate an image regression. At some point the image regression will be fixed and we should move up to `latest`

**Comment by danieljurek** [comment]
Steps to validate: 

1. Use `latest` version in the image configuration
2. Run this pipeline: [`azure-dev - template - tests`](https://dev.azure.com/azure-sdk/internal/_build?definitionId=5002)
3. Ensure that the image reported in the pipeline is the expected image 

If you see failures like this, things are still broken: 

```
  1) [chromium] › todo.spec.ts:4:1 › Create and delete item test ===================================

    browserType.launch: 
    ╔══════════════════════════════════════════════════════╗
    ║ Host system is missing dependencies to run browsers. ║
    ║ Missing libraries:                                   ║
    ║     libgobject-2.0.so.0                              ║
    ║     libglib-2.0.so.0                                 ║
    ║     libnss3.so                                       ║
    ║     libnssutil3.so                                   ║
    ║     libsmime3.so                                     ║
    ║     libnspr4.so                                      ║
    ║     libatk-1.0.so.0                                  ║
    ║     libatk-bridge-2.0.so.0                           ║
    ║     libcups.so.2                                     ║
    ║     libgio-2.0.so.0                                  ║
    ║     libdrm.so.2                                      ║
    ║     libdbus-1.so.3                                   ║
    ║     libexpat.so.1                                    ║
    ║     libxcb.so.1                                      ║
    ║     libxkbcommon.so.0                                ║
    ║     libX11.so.6                                      ║
    ║     libXcomposite.so.1                               ║
    ║     libXdamage.so.1                                  ║
    ║     libXext.so.6                                     ║
    ║     libXfixes.so.3                                   ║
    ║     libXrandr.so.2                                   ║
    ║     libgbm.so.1                                      ║
    ║     libpango-1.0.so.0                                ║
    ║     libcairo.so.2                                    ║
    ║     libasound.so.2                                   ║
    ║     libatspi.so.0                                    ║
    ║     libgcc_s.so.1                                    ║
    ╚══════════════════════════════════════════════════════╝

```

**Comment by mikeharder** [comment]
Verified fixed as of image version 66.0.0: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1866678&view=results
